censys/search/v2/api.py

```diff
@@ -261,6 +247,40 @@ def view(
 
         return self._get(self.view_path + document_id, args)["result"]
 
+    def bulk_view(
```

Comment:
When it comes to concurrent stuff like this, I'm always a big fan of letting the user override the number of threads they want to use. Certainly not a blocker, but might want to consider adding in a `max_workers=20` default parameter.

Reply:
Added in ead5e76